### PR TITLE
fix: Disable buy on rmrk/ksm

### LINKED ISSUE #9157: Disable buy on rmrk/ksm
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

Since we fixed transfer function on frontend i havent realized that balances transfer is also checked on the indexer side so buys are not replicated

### Please reproduce in steps

Try to buy anything on the marketplace

### Expected Behavior

You should not be able to buy nfts, until it is resolved

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

O

### Wallet Address & Transaction hash

_No response_

### Relevant log output

_No response_

### Payment link for reward

https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix
  
  ## Context

  - [x] Closes #9157

image

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

File: components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue [hunk 1]
```diff
@@ -81,6 +81,7 @@ import { type AutoTeleportAction } from '@/composables/autoTeleport/types'
 import { type AutoTeleportActionButtonConfirmEvent } from '@/components/common/autoTeleport/AutoTeleportActionButton.vue'
 import { useBuySupportFee } from '@/composables/transaction/utils'
 
+const DISABLED_PURCHASE_PREFIX = ['ksm', 'rmrk']
```
Comment (roiLeo) [inline components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue:84]: Might use isRemark constant instead

**Review by roiLeo — APPROVED** [review]

File: components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue (hunk 1 above)
Comment (Jarsen136) [reply components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue]: ok

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 6.73 USD/DOT ~ 1.486 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x1d8f76ae4b962471a50c579e4bafbf076911f96b48ed053933ab2a44e3022ccc](https://polkadot.subscan.io/extrinsic/0x1d8f76ae4b962471a50c579e4bafbf076911f96b48ed053933ab2a44e3022ccc)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by roiLeo** [comment]
I would show why it's disable in a follow up

**Comment by vikiival** [comment]
> I would show why it's disable in a follow up

Please do open an issue ^-^

**Pull request merged**